pyramid_oereb/lib/renderer/extract/templates/xml/multilingual_text.xml

```diff
@@ -1,5 +1,7 @@
 <%page args="text"/>
 <%
+    ## TODO:this code gives an error when no Text attribute is available i.e when there is no translation
```

Comment:
In which circumstances can it be, that no text attribute is available at this point of code?

Reply:
If translation text is missing the xml gives an error if called in the browser (I think the function which tries to create the dictionary with the local text has a problem if there is no text value). I opened a new issue (#455) for this because for me this is not in the scope of this PR. However it is something that should be keep in mind.

Reply:
But from the model it cant be that there is no text I think? @kdeininger What do you think?

Reply:
Ok But You can close this. The other thing is more an enhancement

Reply:
That I do not know, I just saw that there were errors when I was trying to test in French for which there were no translations available. I cloud build everything with out error but when calling the xml it crashed. For the json there are already some warnings built in:
https://github.com/camptocamp/pyramid_oereb/blob/77a5962c20d0e5c2630bb9abdc72c5bd3de7c837/pyramid_oereb/lib/renderer/extract/json_.py#L147
so there is no problem for the json.